src/test/java/com/rackspace/ceres/app/config/GranularityValidatorTest.java

```diff
@@ -0,0 +1,67 @@
+package com.rackspace.ceres.app.config;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.rackspace.ceres.app.config.DownsampleProperties.Granularity;
+import com.rackspace.ceres.app.config.configValidator.ConcreteGranularityValidator;
+import java.time.Duration;
+import java.util.LinkedList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+
+public class GranularityValidatorTest {
+
```

Comment:
"Each timeslot width should be a multiple of the largest granularity and should also be a multiple of its neighbors."

I think you should test with at least 3 values in the list also.

You should test: 2,4,16 => true
You should test: 2,8,20 => false

Reply:
oh my bad. I thought I had added one with 3. Looks like I had tested locally with 3 in a config file and forgot to add that to the tests